### ISSUE TITLE
Platform vs Evoq

### ISSUE THREAD

**Issue by jncraig** [DNNCommunity/DNNDocs#355]
## Description of documentation change

References to features or other things that exist in Evoq should be identified as specific to Evoq.  All other items by default should be about Platform.  If not specified explicity, the default Platform reference should be mention prominently. if not often.

## Existing documentation URL (if applicable)
In particular, I was looking for information about Connectors.  The only pages refer to Evoq connectors, and the screen shot is an Evoq screen shot, but this should be mentioned in text and/or tags.

DNN 9.3.2 has connectors now, so the documentation should reflect it.

## Existing documentation section (if applicable)
Connectors is where I found it.  I don't know if there are other places

## Proposed area for new documentation (if applicable)
Should stay in the Connector area, just expanded and/or rewritten.

## Additional context
Add any other context about the documentation request that may be helpful with implementation.

Should be identified as Evoq and DNN Version as this doesn't (I think) exist in earlier versions.

**Comment by thabaum** [comment]
Does there need to be any mention of Evoq here?  I would think they would cover all those features on the DNN Evoq Docs which has all the latest Evoq setting and features.

DNNDOCS should be IMO strictly DNN Platform.

**Comment by valadas** [comment]
I agree, Dnn being MIT and knowing there are some other projects being built on top of it, we should focus on Dnn as many of the contributors here do not have those other products to even document them.

**Comment by dnndev** [comment]
While we shouldn't endeavor to document Evoq, I think it is necessary to identify it and other commercial variants that may be developed as such, so that new users to the ecosystem that may have heard these other names can gain some clarity.
